src/pages/resultsView/plots/PlotsTabUtils.tsx

```diff
@@ -26,19 +26,32 @@ import {CoverageInformation} from "../ResultsViewPageStoreUtils";
 import {IBoxScatterPlotData} from "../../../shared/components/plots/BoxScatterPlot";
 import {AlterationTypeConstants, AnnotatedMutation, AnnotatedNumericGeneMolecularData} from "../ResultsViewPageStore";
 import numeral from "numeral";
-import {getUniqueSampleKeyToCategories} from "../../../shared/components/plots/TablePlotUtils";
 import {getJitterForCase} from "../../../shared/components/plots/PlotUtils";
+import {isSampleProfiled} from "../../../shared/lib/isSampleProfiled";
 
 export const CLIN_ATTR_DATA_TYPE = "clinical_attribute";
 export const dataTypeToDisplayType:{[s:string]:string} = {
-    "COPY_NUMBER_ALTERATION": "Copy Number",
-    "MRNA_EXPRESSION": "mRNA",
-    "PROTEIN_LEVEL": "Protein Level",
-    "METHYLATION": "DNA Methylation",
+    [AlterationTypeConstants.MUTATION_EXTENDED]: "Mutation",
+    [AlterationTypeConstants.COPY_NUMBER_ALTERATION]: "Copy Number",
+    [AlterationTypeConstants.MRNA_EXPRESSION]: "mRNA",
+    [AlterationTypeConstants.PROTEIN_LEVEL]: "Protein Level",
+    [AlterationTypeConstants.METHYLATION]: "DNA Methylation",
     [CLIN_ATTR_DATA_TYPE]:"Clinical Attribute"
 };
 
-export const dataTypeDisplayOrder = [CLIN_ATTR_DATA_TYPE, "MRNA_EXPRESSION", "COPY_NUMBER_ALTERATION", "PROTEIN_LEVEL", "METHYLATION"];
+export const mutationTypeToDisplayMutationType:{[oncoprintMutationType:string]:string} = {
```

Comment:
mutationTypeToDisplayName?   or could we use a StringEnum?

Reply:
DisplayName sounds good